m2e-parent/pom.xml

```diff
@@ -120,6 +120,7 @@
 								<ignore>META-INF/ECLIPSE_.SF</ignore>
 								<ignore>jars/*.jar</ignore>
 								<ignore>Eclipse-SourceReferences</ignore>
```

Comment:
Actually, even those shouldn't be ignored as they're supposed to be stable across rebuilds. And if they're not stable, then they should be made stable.

Reply:
Unfortunately they are not stable since they contain the ID of the commit from which the bundle was build.

Reply:
If configured correctly, the referenced commit is the last commit that has affected the particular module; which is stable as long as the bundle is not changed.

Reply:
That's a point.
But I likely added this because it failed the build, but if it would work without ignoring that, it would be even better.